### DISCUSSION TITLE
关于标签云tags和教程示例不一致的问题

### DISCUSSION THREAD

**Discussion by simpnohouse** [YunYouJun/hexo-theme-starry#187] [Q&A 疑难杂症] [answered]
很喜欢云游君的主题，但是我在按照教程安装时产生了一些我无法解决的问题：
在安装hexo-generato-tag并新建标签页，改好index.md后，发现我的标签云和展示的不一致。
![0(ZK3O7%DO8RZ} 6M 58B`X](https://user-images.githubusercontent.com/73152110/137439819-0714f397-e990-4470-9238-6840611cd965.png)
![FBCKI_SSR3YT37IQY16XU7W](https://user-images.githubusercontent.com/73152110/137439829-a37973b1-4fca-47a2-a359-9ee2d2255878.png)
安装hexo-generato-tag反应如下
![image](https://user-images.githubusercontent.com/73152110/137439984-1383f0e1-b948-43f4-acde-96f4b76033ea.png)
是否是安装失败的原因导致的，亦或是其他的原因呢？

**Comment by simpnohouse** [comment]
顺带一提，几乎相同操作的categories分类页和示例一致。。

**Comment by YunYouJun** [comment, marked as answer]
Did you enable `wordcloud`? Just set this to false in `yun.yml`.

```yml
wordcloud:
  enable: false
```

> https://yun.yunyoujun.cn/guide/additional-package-support.html#%E6%A0%87%E7%AD%BE%E4%BA%91-%E8%AF%8D%E4%BA%91

**Reply by simpnohouse** [reply]
ty！
